video/cloud-client/analyze/video_detect_logo.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+
+# [START video_detect_logo]
+
+from google.cloud import videointelligence
+
+
+def detect_logo(local_file_path="resources/googlework_tiny.mp4"):
+    """Performs asynchronous video annotation for logo recognition on a local file."""
+
+    client = videointelligence.VideoIntelligenceServiceClient()
+
+    with io.open(local_file_path, "rb") as f:
+        input_content = f.read()
+    features = [videointelligence.enums.Feature.LOGO_RECOGNITION]
+
+    operation = client.annotate_video(input_content=input_content, features=features)
+
+    print(u"Waiting for operation to complete...")
+    response = operation.result()
+
+    # Get the first response, since we sent only one video.
+    annotation_result = response.annotation_results[0]
+    # Annotations for list of logos detected, tracked and recognized in video.
+    for logo_recognition_annotation in annotation_result.logo_recognition_annotations:
+        entity = logo_recognition_annotation.entity
+        # Opaque entity ID. Some IDs may be available in [Google Knowledge Graph
+        # Search API](https://developers.google.com/knowledge-graph/).
+        print(u"Entity Id : {}".format(entity.entity_id))
+        # Textual description, e.g. `Google`.
```

Comment:
Can delete

Reply:
Unclear what to do here - it seems the "description" field is used in the output and in the test? Are oyu just suggesting to delete the entity ID?

Reply:
Line 47, the comment.

Reply:
Discussed offline - handled in recent commit